### PR TITLE
Added Good First Issues Tool In The Open Source Tools Subcategory 

### LINKED ISSUE #1330: [Add] Add good first issues link to the open source tools
### Site URL

https://goodfirstissues.com/

### Description of the above provided site

This website helps you filter out good first issues according to your tech stack.

### Category

Open Source

### Sub Category

issue tracker

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by tayyab-ilyas**
## Changes proposed

- Added the good first issue tracker tool in the opensource tools subcategory.

## Screenshots

linksHub1

### CODE REVIEW

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made a suggestion on how to improve your description @tayyab-ilyas! :)

File: database/open_source/tools.json [hunk 1]
```diff
@@ -33,5 +33,12 @@
     "url": "https://geeksrepos.com/",
     "category": "open source",
     "subcategory": "open-source-tools"
+  },
+  {
+    "name":"Good First Issues",
+    "description":"Looking for good first issues according to your tech stack? This website will help you find them.",
```
[the remark this answers is not in the document]
Comment (tayyab-ilyas) [reply database/open_source/tools.json]: thank you @CBID2

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Comment by tayyab-ilyas** [comment]
hey @rupali-codes , can you review this? thanks!

**Review by k-deepak04 — APPROVED** [review]

**Pull request merged**